cves/CVE-2012-0858.yml

```diff
@@ -132,7 +124,7 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: The Project had many big changes as the vcc was one of the first commits in 2005, and the fix was in 2011 and 2012. This issue was in this file for a while, but this file is relatively untouched for how long the time period is. Most of the commits are changing documentation and variable names ,
```

Comment:
What about the people? Lots of different people? New features? Show me some evidence that you looked at the commits between the vcc and fix.